packages/foreman/foreman-proxy/foreman-proxy.spec

```diff
@@ -32,8 +32,8 @@ Requires:      %{?scl_prefix_ruby}ruby(release)
 Requires:       foreman-debug
 Requires:       %{?scl_prefix_ruby}rubygems
 Requires:       %{?scl_prefix_ruby}rubygem(rake) >= 0.8.3
-Requires:       %{?scl_prefix_ror}rubygem(sinatra)
-Requires:       %{?scl_prefix_ror}rubygem(rack) >= 1.1.0
+Requires:       %{?scl_prefix}rubygem(sinatra)
```

Comment:
If you keep these changes out of it and only add packages, repoclosure should pass. I'd recommend doing that as part of the last step.

Reply:
Hrm, it actually failed repoclosures without these changes in, is there something else that can be done to avoid repoclosures then?